barman/compression.py

```diff
@@ -600,6 +616,49 @@ def validate(self, pg_server_version, remote_status):
                 "backup_compression_level %d unsupported by "
                 "pg_basebackup compression %s" % (self.config.level, self.config.type)
             )
+        if self.config.workers is not None:
+            issues.append(
+                "backup_compression_workers is not compatible with compression %s"
+                % self.config.type
+            )
+        return issues
+
+
+class ZSTDPgBaseBackupCompressionOption(PgBaseBackupCompressionOption):
+    def validate(self, pg_server_version, remote_status):
+        """
+        Validate zstd-specific options.
+
+        :param pg_server_version int: the server for which the
+          compression options should be validated.
+        :param dict remote_status: the status of the pg_basebackup command
+        :return List: List of Issues (str) or empty list
+        """
+        issues = super(ZSTDPgBaseBackupCompressionOption, self).validate(
+            pg_server_version, remote_status
+        )
+        # "backup_location = server" requires pg_basebackup >= 15
```

Comment:
This comment seems out of place here given the code is not validating anything related to `backup_location`.

Reply:
Indeed, the same thing exist with lz4. I will replace it with:
zstd compression requires pg_basebackup >= 15
same with lz4